syntax = "proto3";

import "hq/dto/Security.proto";
option java_package = "com.huasheng.quant.open.sdk.protobuf.hq.request";
option java_outer_classname = "TickerRequestProto";
message TickerRequest {
	 Security security = 1;
	 int32 limit = 2;//返回的逐笔个数,实际返回数量不一定会返回这么多,最多返回1000个
}
